eyeshade/workers/ads.js

```diff
@@ -0,0 +1,48 @@
+const cron = require('cron-parser')
+const uuidV4 = require('uuid/v4')
+const underscore = require('underscore')
+
+const createPayoutReportQuery = `insert into payout_reports_ads (id) values ($1)`
+
+const selectWalletBalancesQuery = `select account_id, balance from account_balances where account_type = 'payment_id'`
+
+const createPotentialPaymentsQuery = `insert into potential_payments_ads (payout_report_id, payment_id, provider_id, amount) values ($1, $2, $3, $4)`
+
+// Takes a snapshot of ad account balances
+// and inserts them into potential_payments
+const monthly = async (debug, runtime) => {
+  const client = await runtime.postgres.connect()
+  // const walletsCollection = runtime.database.get('wallets', debug)
+  const walletsCollection = await runtime.database.collection('wallets')
```

Comment:
```js
// const walletsCollection = runtime.database.get('wallets', debug)
const walletsCollection = await runtime.database.collection('wallets')
```
I think the commented out line is the one I want to use and that the current one will fail in prod.  However, I don't know how to pass a runtime object with a database object that would result in the wallets collection being defined in the test [here](https://github.com/brave-intl/bat-ledger/pull/610/files#diff-70cf2a668842f4ac48e3542aaff8cd2cR48).

Reply:
create a runtime-database object or use the boot-runtime module like i do here: https://github.com/brave-intl/bat-ledger/pull/595/files#diff-ff2899c03c9e8e31a9b69d7621e08c83R264

Reply:
https://github.com/brave-intl/bat-ledger/blob/master/test/ledger/surveyor.integration.test.js#L23

Reply:
force